server/src/main/java/io/crate/expression/scalar/systeminformation/PgGetSerialSequenceFunction.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.systeminformation;
+
+import io.crate.data.Input;
+import io.crate.expression.scalar.ScalarFunctionModule;
+import io.crate.metadata.FunctionName;
+import io.crate.metadata.NodeContext;
+import io.crate.metadata.Scalar;
+import io.crate.metadata.TransactionContext;
+import io.crate.metadata.functions.Signature;
+import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
+import io.crate.types.DataTypes;
+
+public class PgGetSerialSequenceFunction extends Scalar<String, Object> {
+
+    public static final String NAME = "pg_get_serial_sequence";
+    private static final FunctionName FQN = new FunctionName(PgCatalogSchemaInfo.NAME, NAME);
+
+    public static void register(ScalarFunctionModule module) {
+        module.register(
+            Signature.scalar(
+                FQN,
+                DataTypes.STRING.getTypeSignature(),
+                DataTypes.STRING.getTypeSignature(),
+                DataTypes.STRING.getTypeSignature()
+            ),
+            PgGetSerialSequenceFunction::new
```

Comment:
Can we use the `BinaryScalar` here instead?
```suggestion
            (signature, boundSignature) -> new BinaryScalar<>(
                (table, column) -> null,
                signature,
                boundSignature,
                DataTypes.STRING
            )
```

Reply:
I was under the impression, that the `BinaryScalar` is mostly intended for arithmetic operations, where both inputs, as well as the output are of the same type <T>. (Which by change is the case here, but still felt somewhat "wrong")

Reply:
> I was under the impression, that the BinaryScalar is mostly intended for arithmetic operations, where both inputs, as well as the output are of the same type .
> Which by chance is the case here, but still felt somewhat "wrong"

Why is it by chance? the signature of this function is `String, String` and return type is also String.

Reply:
But no strong opinion here, would just remove boilerplate code.

Reply:
I switched it to `BinaryScalar`